engine-transactions/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "aurora-engine-transactions"
+version = "1.0.0"
+authors = ["NEAR <hello@near.org>"]
+edition = "2018"
+description = ""
+documentation = ""
+readme = true
+homepage = "https://github.com/aurora-is-near/aurora-engine"
+repository = "https://github.com/aurora-is-near/aurora-engine"
+license = "CC0-1.0"
+publish = false
+autobenches = false
+
+[dependencies]
+aurora-engine-types = { path = "../engine-types", default-features = false }
+aurora-engine-sdk = { path = "../engine-sdk", default-features = false }
+aurora-engine-precompiles = { path = "../engine-precompiles", default-features = false }
+evm = { git = "https://github.com/aurora-is-near/sputnikvm.git", default-features = false }
+rlp = { version = "0.5.0", default-features = false }
+hex = { version = "0.4", default-features = false, features = ["alloc"] }
+
+[features]
+std = ["aurora-engine-types/std", "aurora-engine-precompiles/std"]
```

Comment:
Probably should also have

```
"evm/std", "rlp/std", "hex/std"
```

assuming each of those crates also have a `std` feature.